### DISCUSSION TITLE
Absolute imports with hot reloading fix question

### DISCUSSION THREAD

**Discussion by gmcamposano** [prisma/prisma#16607] [Q&A] [answered]
```
// dbconnect.js

import { PrismaClient } from '@prisma/client'
let global = PrismaClient || undefined
export let prisma =
  global.prisma ||
  new PrismaClient({
    log: ['query']
  })
if (process.env.NODE_ENV !== 'production') global.prisma = prisma
```
I have set in NextJS the absolute path config:
```
{
  "compilerOptions": {
    "baseUrl": "."
  }
}
```
Before setting up the absolute path, whenever i used:
import {prisma} from '/prisma/dbconnect'  => I got autocomplete working with prisma
Now, because of using the absolute path config, whenever i use prisma i get no autocomplete. Am i doing something wrong here?

**Comment by gmcamposano** [comment, marked as answer]
nevermind:
Solution is to add:  `import { prisma } from 'prisma/dbconnect'`
NOT: `import { prisma } from '/prisma/dbconnect'`

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
